pkg/ocicni/types.go

```diff
@@ -64,7 +65,7 @@ type PodNetwork struct {
 	// NetworkConfig is configuration specific to a single CNI network.
 	// It is optional, and can be omitted for some or all specified networks
 	// without issue.
-	NetworkConfig map[string]NetworkConfig
+	RuntimeConfig map[string]RuntimeConfig
```

Comment:
In scenario for CRI, we assume that CRI does not know about details of default network but only use it. So how can CRI put bandwidth and portmappings into this map, what should the **key** be? default?

Reply:
Does we need a DefaultRuntimeConfig for pod coming from CRI?

Reply:
If you have a PodCIDR or other information that you'd like to apply to the default network, then you can do something like podman does:

https://github.com/containers/libpod/pull/1623/commits/5c5163107cb31d83eb6cd05427e818182c1f6112#diff-a35062722abcf8eead2a8bff75594fa3R29

I believe this approach was chosen because it was less confusing than overloading "" to mean the default network in the NetworkConfig/RuntimeConfig array (because the default network name could also be used, so then you might have two entries there and which one do you pick...)

Reply:
OK, but there maybe some troubles on downwards compatibility and default network change after getting, anytime, its an elegant way to support both CRI and podman. I'll rebase my #34 PR later based on your commit. Thanks ~